src/api/api.ts

```diff
@@ -0,0 +1,2 @@
+export { MetaPointer, readMetaPointerFromFile } from './metapointer.js';
```

Comment:
_This comment is about the whole file._

We could move this module to `src/index.ts`, and we are all set about API.